dbms/src/Storages/MergeTree/BackgroundProcessingPool.cpp

```diff
@@ -180,6 +181,7 @@ void BackgroundProcessingPool::threadFunction()
         catch (...)
         {
             tryLogCurrentException(__PRETTY_FUNCTION__);
+            DNSCacheUpdater::incrementNetworkErrors();
```

Comment:
Is it unfinished?
Why we increment network errors here?

Reply:
Background tasks [here](https://github.com/yandex/ClickHouse/pull/2141/files#diff-6d7b8a71e9cf645fa1e86f3e40f7ff25R178) could be affected by network errors also (for example fetching data from outdated IP address).
So, we need to count them also.